webapp/packages/core-root/src/ServerNodeService.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { Bootstrap, injectable } from '@cloudbeaver/core-di';
+import { Executor, IExecutor } from '@cloudbeaver/core-executor';
+import { GraphQLService, WsSocketConnectedEvent } from '@cloudbeaver/core-sdk';
+
+import { ServerNodeError } from './ServerNodeError';
+import { ServerEventId, SessionEventSource } from './SessionEventSource';
+
+@injectable()
+export class ServerNodeService extends Bootstrap {
+  applicationRunId = '';
+
+  onApplicationRunIdChange: IExecutor;
+  constructor(
+    private readonly graphQLService: GraphQLService,
+    private readonly sessionEventSource: SessionEventSource
+  ) {
+    super();
+    this.onApplicationRunIdChange = new Executor();
+    this.sessionEventSource.onEvent<WsSocketConnectedEvent>(ServerEventId.CbSessionWebsocketConnected, data => {
+      this.applicationRunIdChanged(data.applicationRunId);
+    });
+  }
+
+  register(): void { }
+
+  load(): void { }
+
+  applicationRunIdChanged(applicationRunId: string): void {
+    if (this.applicationRunId === '') {
```

Comment:
I would suggest using "null" as the default value instead of an empty string or any other value if you want to clearly indicate that no value has been set. This will help avoid any confusion or unexpected behavior that might arise if an empty string or other default value is interpreted as a valid input.

Reply:
Fixed this